paasta_itests/steps/setup_marathon_job_steps.py

```diff
@@ -51,14 +52,14 @@
             {'hostPath': u'/nail/srv', 'containerPath': '/nail/srv', 'mode': 'RO'},
             {'hostPath': u'/etc/boto_cfg', 'containerPath': '/etc/boto_cfg', 'mode': 'RO'}]
     },
-    'instances': 1,
+    'instances': 3,
```

Comment:
I'm surprised this doesn't screw up other tests. If possible let this stay at 1 and for your needs adjust it as necissary